lib/mumuki/laboratory/locales/es-CL.yml

```diff
@@ -11,14 +11,16 @@ es-CL:
   and: y
   appendix: Apéndice
   appendix_teaser: ¿Quieres saber más? <a href="%{link}">Consulta el apéndice de este capítulo</a>
-  are_you_sure: '¿Estás seguro que quieres %{action}?'
-  ask_a_question: ¡Haz una pregunta!
+  approved_message: Validado por mentor
```

Comment:
Although this translation predates this PR, I don't like the fact that it is gender-biased. Perhaps we could use _Validado por mentorías_. We should check that with @Rgonzalezt or @lauramangifesta

Reply:
Yes, i like that suggestion. We could use "Validado por mentorías" instead of "Validado por mentor".